packages/web-twig/src/Resources/components/Toast/README.md

```diff
@@ -0,0 +1,284 @@
+| `default`     | `info`        |
+| `informative` | `info`        |
+| `success`     | `check-plain` |
+| `warning`     | `warning`     |
+
+### Action Link
+
+An action link can be added to the ToastBar component:
+
+```twig
+<ToastBar>
+  Message with action
+  <Link href="#" color="inverted" isUnderlined>Action</Link>
+</ToastBar>
+```
+
+👉 For the sake of flexibility, developers can pass the link as part of the message. However, it is strongly recommended
+to use the **inverted underlined** variant of the link to make it stand out from the message.
+
+👉 **Do not put any important actions** like "Undo" in the ToastBar component (unless there are other means to perform
+said action), as it is very hard (if not impossible) to reach for users with assistive technologies. Read more about
+[Toast accessibility](#scott-o-hara-toast) at Scott O'Hara's blog.
+
+### Colors
+
+The ToastBar component is available in all [emotion colors][dictionary-color], plus the `inverted` variant (default).
+Use the `color` option to change the color of the ToastBar component.
+
+For example:
+
+```twig
+<ToastBar color="success">
+  Success message
+</ToastBar>
+```
+
+### Opening the ToastBar
+
+Use our JavaScript plugin to open a Toast **that is present in the DOM,** e.g.:
+
+```twig
+<Button
+  data-spirit-toggle="toast"
+  data-spirit-target="#my-hidden-toast"
+  aria-expanded="false"
+>
+  Show the hidden toast
+</Button>
+```
+
+👉 Advanced toast queue control is not yet implemented.
+
+### Dismissible ToastBar
+
+To make the ToastBar dismissible, add the `isDismissible` prop along with a unique `id` attribute:
+
+```twig
+<ToastBar id="my-dismissible-toast" isDismissible>
+  Dismissible message
+</ToastBar>
+```
+
+### API
+
+| Name            | Type                                                         | Default    | Required | Description                                                          |
+| --------------- | ------------------------------------------------------------ | ---------- | -------- | -------------------------------------------------------------------- |
+| `color`         | [[Emotion Color dictionary][dictionary-color] \| `inverted`] | `inverted` | ✕        | Color variant                                                        |
+| `closeLabel`    | `string`                                                     | `Close`    | ✕        | Close label                                                          |
+| `hasIcon`       | `bool`                                                       | `false` \* | ✕        | If true, an icon is shown along the message                          |
+| `iconName`      | `string`                                                     | `info` \*  | ✕        | Name of a custom icon to be shown along the message                  |
+| `id`            | `string`                                                     | —          | ✕        | Optional ToastBar ID. Required when `isDismissible` is set to `true` |
```

Comment:
"required" 

is it? Will it break?

Reply:
(the JS probably will, but it will still render right?)

Reply:
Just like that: rendering will be OK, interaction broken. I could not find a way to treat it in Twig…